contracts/VeBalDelegator.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.11;
+
+import "./VoteEscrowTokenManager.sol";
+import "./LiquidityGaugeVoter.sol";
+import "./GovernorVoter.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../interfaces/balancer/ILiquidityGauge.sol";
+import "../interfaces/balancer/ILiquidityGaugeController.sol";
+import "../interfaces/balancer/IVeToken.sol";
+import "../interfaces/notional/IStakedNote.sol";
+
+/// @title 80-BAL-20-WETH BPT PCV Deposit
+/// @author Fei Protocol
+contract VeBalDelegator is
+    VoteEscrowTokenManager,
```

Comment:
Was there a snapshot delegator in the original Fei code? Balancer does a lot of voting on snapshot so we might want to make sure we can delegate for that:

https://docs.snapshot.org/guides/delegation

Reply:
yeah, I added the snapshot delegator back.